operator_ui/src/components/BaseLink.test.tsx

```diff
@@ -1,29 +1,41 @@
 import React from 'react'
-import render from '../../support/test-helpers/renderWithinRouter'
+import { MemoryRouter } from 'react-router-dom'
+import { render, screen } from '@testing-library/react'
 import BaseLink from '../../src/components/BaseLink'
 
+const { queryByText } = screen
+
+const renderBaseLink = (link: React.ReactNode) =>
+  render(<MemoryRouter>{link}</MemoryRouter>)
+
 describe('components/BaseLink', () => {
   it('renders an anchor', () => {
-    const component = render(<BaseLink href="/foo">My Link</BaseLink>)
-    expect(component.text()).toContain('My Link')
-    expect(component.prop('href')).toEqual('/foo')
+    renderBaseLink(<BaseLink href="/foo">My Link</BaseLink>)
+
+    expect(queryByText('My Link')).toBeInTheDocument()
```

Comment:
Should be `getByText` if we don't expect it to be `undefined`. This is a good cheat sheet - https://testing-library.com/docs/dom-testing-library/cheatsheet/#queries